XenModel/Messages.resx

```diff
@@ -2651,6 +2651,9 @@ Do you want to assign it to the schedule '{2}' instead?</value>
   <data name="CHECKING_HA_STATUS" xml:space="preserve">
     <value>Checking HA and WLB status</value>
   </data>
+  <data name="CHECKING_HEALTH_CHECK_SERVICE" xml:space="preserve">
+    <value>Checking pool health check service status</value>

```

Comment:
I think `Checking pool enrollment to the Health Check service` is more accurate.

Reply:
Fixed.